src/SnehalW/ArratContainNumber.java

```diff
@@ -0,0 +1,23 @@
+package SnehalW;
+
+public class ArratContainNumber
+{
+
```

Comment:
Hi @SnehalWAug19 
Here Logic is wrong...
here you are checking the not condition.... if that number not exist in array you breaking the for loop but... check for the any number which is in array, For that your logic will be wrong...

you should check the if that number exist in array then fine(do not print or break) but if not (else) condition you need to print and break that condition...

Hope your getting my point...